lib/src/ruby/module.cc

```diff
@@ -883,7 +883,16 @@ namespace facter { namespace ruby {
                 raise = true;
                 fail_option = ruby.nil_value();
             }
-            return execute_command(ruby.to_string(argv[0]), fail_option, raise, timeout);
+            bool expand = true;
+            try{
+            volatile VALUE expand_value =  ruby.to_symbol("expand");
+            volatile VALUE expand_option = ruby.rb_hash_lookup3(argv[1], ruby.nil_value(), expand_value);
```

Comment:
Maybe not the best way to pass new expand parameter, but leatherman does not compile with this option